user_research/Trashes/trash_recyclingCenter.md

```diff
@@ -10,43 +15,91 @@ We can talk about trashes in three different ways:
+    
+The **rush hour** of Ustom's RC are at 10 am and 3 pm [D's I]
+
+**Time spend in RC** [D's I]:
+ * Users spend around **5-6 min inside** the RC
+ * During rush hours, people are **waiting aroud 15 minutes** at the entry 
+  * *Personal consideration : Perfect to place an ad about 6element*
+  
+In summer, bin are usually full and **people must come back at home with trashes** : *"It would be great to have the information in live"* [d's I]
+
+Bins are undersized
+
+Summer increases the number of each trashes, not only green ones
+
+
+
 # Users
-* **Any students** use recycling center [doc LV]
-* 4,000 people lives in a specific area. 5,000 users. 96,000 visits a year [doc LV]
+**Students** usually don't use recycling center [doc LV]
+
+Example for a given recycling center: 
+* 40,000 people live in the area. 5,000 users. 96,000 visits a year [doc LV]
   * a user comes **19 times a year** in the recycling center
-  * **3/4** residents don't use them. 
  * *Personal consideration* : It would be more relevant to talk about home instead of people
-* People leaving in detached house with garden are the **main users** of recycling center [Vitry's interview]
-* Urbans don't come in recycling center, they throw out their trashes on the street. [V's I]
-* Public recycling centers are not made for **professionals** who are supposed to use private professional recycling centers. There are 5-6 in this kind in Gironde. Some local authority accept them in recycling centers but *try* to make them pay. [LV's I]
 
-![Professional](professional.png)
-* Punctual users: punctual events such as a move or spring clean. [doc LV]
-* Bigger the place we live in is, more trash we can accumulate before throwing out everything at the same time. People living in tiny places can't keep any trash and have to throw them out immediately. [doc LV]
+People leaving in detached house with garden are the **main users** of recycling center [Vitry's interview]
+
+Ustom's RC are base in country area so there mainly country persons[D's I]
+
+There are a lot of **retired person** coming and almost **every day** to have **social interaction**: they usually throw out some bottles. These persons represent a little percentage but they come around **200 times a year** [D's I]
+**Retired person** represent more than **50%** of RC's users [D's I]
+
+Users come between **1 and 5 times a year** in RC [D's T]
+
+The user which is the most representative is **family's father coming after spring-clean**
+
+There are a lot of **farmers**, wine-makers, ... in Aquitaine so they represent a big part of users [Delphine's interview]
+
+**Urbans** don't come in recycling center, they throw out their trashes on the street. [V's I]
+
+Recycling centers mastered by Ustom guest particular and professional users [D's I]
+* *Visit* :For 100 visitors, there are around **70 particulars for 30 professionals**
+* *Users* : RC count **22,000 particulars for 1,000 professionals**
```

Comment:
The difference is about visit frequency : there are 1/22 of RC users who are professionals but for 100 visits (it's a mean) , we can count 30% of professionals. So there are not a lot of professionals registered in comparison with private individuals but they come more often. Am I clear ? How do you think I should rephrase it?